ext/calendar/jewish.c

```diff
@@ -433,16 +433,25 @@ static void MoladOfMetonicCycle(
 								   zend_long *pMoladHalakim)
 {
 	register zend_ulong r1, r2, d1, d2;
+	zend_long chk;
 
 	/* Start with the time of the first molad after creation. */
 	r1 = NEW_MOON_OF_CREATION;
+	chk = (zend_long)metonicCycle;
```

Comment:
Note that if zend_long is 32-bit longs, you still have an issue.
Furthermore, I think the failure should be propagated instead of silently set to 0.

Reply:
> Furthermore, I think the failure should be propagated instead of silently set to 0.

That sounds more a change for master ?